packages/flutter/lib/src/cupertino/activity_indicator.dart

```diff
@@ -128,13 +143,16 @@ class _CupertinoActivityIndicatorPainter extends CustomPainter {
     canvas.save();
     canvas.translate(size.width / 2.0, size.height / 2.0);
 
+    // Rotate so that the top tick is the first to be drawn.
+    canvas.rotate(math.pi / 2);
+
     final int activeTick = (_kTickCount * position.value).floor();
 
-    for (int i = 0; i < _kTickCount; ++ i) {
+    for (int i = ((_kTickCount - 1) * progress).toInt(); i >= 0; --i) {
```

Comment:
If I remember correctly https://developer.apple.com/design/resources/ actually has both clockwise and anticlockwise versions. But in xcode I'm only seeing the clockwise version. Thank you for catching this!